Added dependency for derived trait not working

```log
error[E0433]: failed to resolve: use of undeclared crate or module `utoipa`
  --> db/src/queries/product.rs:49:53
   |
49 | #[derive(serde::Serialize, Debug, Clone, PartialEq, utoipa::ToSchema)]
   |                                                     ^^^^^^ use of undeclared crate or module `utoipa`

For more information about this error, try `rustc --explain E0433`.
error: could not compile `db` (lib) due to 1 previous error
```

I already add dependencies to the `build.rs`.
```rs
use std::collections::HashMap;
use std::env;
use std::path::PathBuf;

use clorinde::config::{Config, Package};
use clorinde::config::{CrateDependency, Types};
use dotenv::dotenv;
use postgres::Client;
use postgres::NoTls;

fn main() {
    dotenv().ok();

    let dependencies = [("utoipa".into(), CrateDependency::Simple("5.3".into()))]
        .into_iter()
        .collect();
    let cfg = Config {
        r#async: true,
        serialize: true,
        queries: PathBuf::from("queries"),
        destination: PathBuf::from("db"),
        package: Package {
            name: "db".into(),
            ..Default::default()
        },
        types: Types {
            crate_info: dependencies,
            mapping: HashMap::new(),
            derive_traits: vec![],
        },
        ..Default::default()
    };
    let url = env::var("DATABASE_URL").expect("DATABASE_URL must be set");
    let mut client = Client::connect(&url, NoTls).unwrap();
    clorinde::gen_live(&mut client, cfg).unwrap();
}

```


```sql
--: Product(description?) : utoipa::ToSchema

--! get_by_id : Product
SELECT
    *
FROM
    products
WHERE
    id = :id
    AND is_active = TRUE;
```

The generated code derived `utoipa::ToSchema`.

```rs
#[derive(serde::Serialize, Debug, Clone, PartialEq, utoipa::ToSchema)]
pub struct Product {
    pub id: i32,
    pub description: Option<String>,
    // ...
}
```

However, the generated `Cargo.toml` missing the `utoipa` crate.
```toml
# This file was generated with `clorinde`. Do not modify.
[package]
name = "db"
version = "0.1.0"
edition = "2021"
publish = false

[features]
default = ["deadpool", "chrono"]
deadpool = ["dep:deadpool-postgres", "tokio-postgres/default"]
wasm-async = ["tokio-postgres/js", "chrono?/wasmbind", "time?/wasm-bindgen"]

chrono = ["dep:chrono"]
time = ["dep:time"]

[dependencies]
## Core dependencies
# Postgres types
postgres-types = { version = "0.2.9", features = ["derive"] }
# Postgres interaction
postgres-protocol = "0.6.8"
# Iterator utils required for working with `postgres_protocol::types::ArrayValues`
fallible-iterator = "0.2.0"

## Types dependencies
# TIME, DATE, TIMESTAMP or TIMESTAMPZ
chrono = { version = "0.4.39", optional = true }
time = { version = "0.3.37", optional = true }
# JSON or JSONB
serde_json = { version = "1.0.134", features = ["raw_value"] }
serde = { version = "1.0.217", features = ["derive"] }

## Async client dependencies
# Postgres async client
tokio-postgres = { version = "0.7.13", features = ["with-chrono-0_4", "with-time-0_3", "with-serde_json-1"] }
# Async utils
futures = "0.3.31"

## Async features dependencies
# Async connection pooling
deadpool-postgres = { version = "0.14.1", optional = true }
```

--- a/clorinde/src/codegen/cargo.rs
+++ b/clorinde/src/codegen/cargo.rs
@@ -184,37 +184,6 @@ pub fn gen_cargo_file(dependency_analysis: &DependencyAnalysis, config: &Config)
         .line("# Iterator utils required for working with `postgres_protocol::types::ArrayValues`");
     cargo.dep("fallible-iterator", DependencyTable::new("0.2.0"));
 
-    // add custom type crates
-    if !config.types.mapping.is_empty() {
-        cargo.line("\n## Custom type crates");
-
-        let references_default_crate = config.types.mapping.values().any(|t| {
-            match t {
-                crate::config::TypeMapping::Simple(t) => t,
-                crate::config::TypeMapping::Detailed { rust_type, .. } => rust_type,
-            }
-            .starts_with("ctypes::")
-        });
-
-        if !config.types.crate_info.is_empty() {
-            let mut crates: Vec<_> = config.types.crate_info.iter().collect();
-            crates.sort_by(|(name_a, _), (name_b, _)| name_a.cmp(name_b));
-
-            for (name, dep) in crates {
-                match dep {
-                    CrateDependency::Simple(version) => {
-                        cargo.line(&format!("{} = \"{}\"", name, version));
-                    }
-                    CrateDependency::Detailed(dependency) => {
-                        cargo.dep(name, dependency.to_owned());
-                    }
-                }
-            }
-        } else if references_default_crate {
-            cargo.line("ctypes = {{ path = \"../ctypes\" }}");
-        }
-    }
-
     let mut client_features = Vec::new();
 
     if dependency_analysis.has_dependency() {
@@ -294,6 +263,23 @@ pub fn gen_cargo_file(dependency_analysis: &DependencyAnalysis, config: &Config)
             "deadpool-postgres",
             DependencyTable::new("0.14.1").optional(),
         );
+    }
+
+    if !config.types.crate_info.is_empty() {
+        cargo.line("\n## Custom type dependencies");
+        let mut crates: Vec<_> = config.types.crate_info.iter().collect();
+        crates.sort_by(|(name_a, _), (name_b, _)| name_a.cmp(name_b));
+
+        for (name, dep) in crates {
+            match dep {
+                CrateDependency::Simple(version) => {
+                    cargo.line(&format!("{} = \"{}\"", name, version));
+                }
+                CrateDependency::Detailed(dependency) => {
+                    cargo.dep(name, dependency.to_owned());
+                }
+            }
+        }
     }
 
     cargo.into_string()

--- a/examples/auto_build/build.rs
+++ b/examples/auto_build/build.rs
@@ -1,6 +1,9 @@
 use std::{path::PathBuf, str::FromStr};
 
-use clorinde::{Error, config::Config};
+use clorinde::{
+    Error,
+    config::{Config, Package},
+};
 
 // This script will generate a new clorinde crate every time your schema or queries change.
 // In this example, we generate the module in our project, but
@@ -13,6 +16,10 @@ fn main() -> Result<(), Error> {
 
     let cfg = Config {
         destination: PathBuf::from_str("auto_build_codegen").unwrap(),
+        package: Package {
+            name: "auto_build_codegen".into(),
+            ..Package::default()
+        },
         ..Default::default()
     };
 

--- a/examples/custom_types/custom_types_codegen/Cargo.toml
+++ b/examples/custom_types/custom_types_codegen/Cargo.toml
@@ -22,10 +22,6 @@ postgres-protocol = "0.6.8"
 # Iterator utils required for working with `postgres_protocol::types::ArrayValues`
 fallible-iterator = "0.2.0"
 
-## Custom type crates
-ctypes = { path = "../ctypes" }
-postgres_range = { version = "0.11.1", features = ["with-chrono-0_4"] }
-
 ## Types dependencies
 # TIME, DATE, TIMESTAMP or TIMESTAMPZ
 chrono = { workspace = true, optional = true }
@@ -42,3 +38,7 @@ futures = "0.3.31"
 ## Async features dependencies
 # Async connection pooling
 deadpool-postgres = { version = "0.14.1", optional = true }
+
+## Custom type dependencies
+ctypes = { path = "../ctypes" }
+postgres_range = { version = "0.11.1", features = ["with-chrono-0_4"] }

